location-proof-protocol.md

```diff
@@ -0,0 +1,77 @@
+#########################
+
+## For review only
+
+#########################
+
+# 📍 Location Proof Protocol
+
+The Location Proof Protocol aims to address the challenge of proving location on the decentralized web in a flexible, yet standardized way.
+
+Given the diverse requirements and approaches for location proofs, our goal is to support innovation across a wide range of proof strategies
+while providing a standard data model for creating and verifying these proofs.
+
+This repository currently contains the code necessary to deploy the core Ethereum Attestation Service (EAS) schema for the protocol. The
+standard includes a framework (in development) for building "recipes" that allow for many different strategies for proving location to be
+added.
+
+## The Challenge
+
+### Broad Approaches to Location Proofs
+
+- **Authority-based Strategies**: Authorized individuals attest to someone's presence. i.e. ticket checkers, event hosts
+- **Social Strategies**: People verify each other's claims, confirming or challenging their validity. i.e. permissionless social
+  confirmation, perhaps with sybil-resistant scoring of confirmer accounts; PIN exchange
+- **Near-field Machine Strategies**: Hardware devices attest to the presence of another device through information exchange or cryptographic
+  'endorsement' of a claim. i.e. RFID; NFC; Bluetooth
+- **Network Machine Strategies**: Nodes on a local network or hardware wallets triangulate position and sign confirmation attestations. i.e.
+  Time of Flight; Time Difference of Arrival
+- **Sensor Data Strategies**: Where devices sense local networks, environmental conditions, images + audio, accelerometers, etc, and
+  location is discerned from analysis of this evidence. i.e. radio frequency / optical / inertial / acoustic / magnetometer localization
```

Comment:
Should this be combined with "Near-field"

Reply:
I see them as different — Near-field is an active scan of another device, sensor data is picking up ambient measurements of the physical and digital environment